### PR TITLE
Comments Section on Form

### LINKED ISSUE #198: Create Comments Section on Form (3)
As a _Student_,
I want to be able to write comments on my form,
So that I can remember important information for myself and others.

- **Acceptance Criteria:**
- There exists a field on the student user interface of the form which allows for text comments to be inputted.

**User Story Points:** 3

### PR DESCRIPTION
**Pull request by spencerj411**
closes #198.

**Warning: this branch is based off feature-plan-save-and-load**
There is now a comments section on the form (no save and load feature included). This required me to create a new StudentModel and a new EditStudentFromSnippet class.

image

Edit:
Had to change a few other things due to PR #235.
Changes include:

- Moved textarea to the persistent sidebar
- Modified student_edit.js to get saving/loading of comments working
- modified style.css to fix the sizing of the textarea (ANU Styling sets textarea's widths to just a bit longer than 100%).
- removed code (from a previous implementation) that was not being used

image

### CODE REVIEW

File: cassdegrees/api/models.py [hunk 1]
```diff
@@ -69,3 +69,8 @@ class ProgramModel(models.Model):
 
     class Meta:
         unique_together = (("code", "year"), ("name", "year"))
+
+
+class StudentModel(models.Model):
```
Comment (j-selby) [inline cassdegrees/api/models.py]: Are we storing this in the main database? From what I understood, this wasn't going to be the case?

File: cassdegrees/templates/student_index.html [hunk 2]
```diff
@@ -32,6 +32,7 @@
 
     <div>
     {% for plan in plans %}
+
```
Comment (j-selby) [inline cassdegrees/templates/student_index.html]: Random space

File: cassdegrees/ui/views/student.py [hunk 3]
```diff
@@ -93,7 +95,7 @@ def student_create(request):
     if id_to_view:
         # Create a new cookie in the default plan location containing compressed relevant plan details
         request.session['plan:'] = compress(
-            {'program_id': int(id_to_view), 'date': timezone.localtime().strftime('%d/%m/%Y %H:%M')}
+            {'program_id': int(id_to_view), 'date': timezone.now().strftime('%d/%m/%Y')}
```
Comment (j-selby) [inline cassdegrees/ui/views/student.py]: Is this part of this PR?

**Review by j-selby — CHANGES_REQUESTED** [review]
Few tweaks needed

**Review by JackSekoranja — APPROVED** [review]
Looks good and works, happy to merge

File: cassdegrees/ui/views/student.py (hunk 3 above)
Comment (JackSekoranja) [reply cassdegrees/ui/views/student.py]: This is the result of a merge with my branch

File: cassdegrees/api/models.py (hunk 1 above)
Comment (spencerj411) [reply cassdegrees/api/models.py]: removed

**Review by JackSekoranja — APPROVED** [review]
Works well and looks good. I did notice that you can expand the text box further than we want the users to, but this isn't a huge issue (See Attached Image). Happy for you to merge

Screen Shot 2019-08-26 at 3 58 23 pm

**Pull request merged**